### DISCUSSION TITLE
Termux Asking Again and again

### DISCUSSION THREAD

**Discussion by Haris2006n** [termux/termux-app#2946] [Q&A] [answered]
Hey I Just want to run a commND ON MY Termux ./bxi but when i write that command a error pops up that 
Warning : ap does not have a stable CLI interface. use with caution in scripts 
honestly i dont have issues with this errors 
then it says that
it appears that directory '~storage already exists 
then few lines
then it asks  do you want to continue y/n 
i put y but it all starts from the warning again and again
![WhatsApp Image 2022-08-22 at 1 28 01 AM](https://user-images.githubusercontent.com/111707803/185809836-488a24fd-0584-4385-babd-3aed9daff94d.jpeg)

**Comment by DethByte64** [comment]
What is bxi supposed to do?

**Reply by Haris2006n** [reply]
facebook ids cloning

**Reply by cujoXavier** [reply]
> facebook ids cloning

🙄 wow ...
Nickels worth of free advice: 
Instead of trying the wrong way to clone social media profile IDs, why not try to learn more about CLI basics? At least read some of the plethora of FREE and open source information, great  for noobs & experts alike-- and get a good handle on CLI, virtual storage systems, containers, subsystems etc. And about running a powerful API & main application in a stock mobile ROM without root access. See if you can find out what you should already know if you are going to be breaking laws for (imo) immature purposes- but that's your business - I simply recommend learning more about the same API and frameworks that are warning you not to do something the way you are repeatedly trying to. I don't condone what you are trying to do - nor do I really care... Also- 1 thing that goes to show me that you are a noob script kiddie with no grit- you have the answer to your issue IN THE SCREENSHOT you posted. But, as Apollo said, (very nicely I might add) I would have told you that you've been warned and next time you will be banned if you bring that or anything else in violation of the rules as spelled out in 
https://wiki.termux.com/wiki/Hacking

"Termux asking again and again"...
Yeah, and you still didn't get what the problem was... not termux! Termux was doing you a favor... warning you off a 3rd party script from the errors in your execution, and probably your installations... 
Learn how to do it the right way, don't take shortcuts, you will definitely miss the sh--tuff that makes you good at the same time as you are learning to be bad ... 
in one eye out the other- at least I tried ..

**Comment by Maxython** [comment]
https://wiki.termux.com/wiki/Hacking

**Comment by agnostic-apollo** [comment, marked as answer]
We don't provide support for third party scripts. Also check https://wiki.termux.com/wiki/Hacking

The storage warning is from `termux-setup-storage` and is expected if running again.

https://wiki.termux.com/wiki/Termux-setup-storage

https://github.com/termux/termux-tools/blob/master/scripts/termux-setup-storage.in
